README.md

```diff
@@ -0,0 +1,27 @@
+# school-library
+this project is ruby application for school library management.
+
+## Authors
+
+👤 **Souad**
+
+- GitHub: [@githubhandle](https://github.com/souad988)
+  https://github.com/souad988  
+- LinkedIn: [@linkedinbandle](https://www.linkedin.com/in/souad-el-mansouri/)  
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!  
+
+Feel free to check the (https://github.com/souad988/school-library/issues).  
+
+Show your support  
+
+Give a ⭐️ if you like this project!  
+
+## ✨ Acknowledgments
+
+-microverse
+
+📝 License
+This project is MIT licensed.
```

Comment:
- [ ] Kindly add the missing sections like the `Built With, Getting Started and how to clone your project, and Show your support`. Please do not forget to personalize it to match this project.